packages/@okta/vuepress-site/.vuepress/scripts/updateBuildScript.js

```diff
@@ -19,15 +17,23 @@ const createClientConfig = require('../webpack/createClientConfig')
 const createServerConfig = require('../webpack/createServerConfig')
 const { applyUserWebpackConfig } = require('../util/index')
 
+const MAX_WORKER_THREADS = 6;
+const PAGES_PER_THREAD = 100;
```

Comment:
We can tune this more for better performance but I'm seeing consistent passes right now so I'm leaving it this way. It appears 8 threads is too many for the VM memory.